core-bundle/src/Security/DataContainer/UpdateAction.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Security\DataContainer;
+
+class UpdateAction extends AbstractAction
+{
+    use CurrentTrait;
+    use NewTrait;
+
+    public function __construct(
+        private string $dataSource,
+        private array $current,
+        private ?array $new = null,
+    ) {
+        parent::__construct($dataSource);
+    }
+
+    protected function getSubjectInfo(): array
+    {
+        $subject = parent::getSubjectInfo();
+        $subject[] = 'ID: '.$this->getCurrentId();
```

Comment:
Should this also include the new ID? Or can the current and new ID not really be different anyway? 🤔